.travis.yml

```
@@ -4,6 +4,8 @@ php:
    - '7.2'
    - '7.3'
    - '7.4'
+   - '8.0'
+   - '8.1'
```

```suggestion
   - '8.1.0'
```

It looks like we need full version atm for 8.1.

See https://travis-ci.community/t/php-8-1-support/12439/23 for details